### PR TITLE
Branch labelled wrong

### PR DESCRIPTION
**Pull request by Nick-Zabalza**
@cmonez @jeremylfchen 

Hey guys,

I intended this PR to be about DB comparison testing but the alterations to my seeding scripts, to acheive 10million records, were such that that ended up being the focus of this PR.

I had to alter the various files and even add completely new ones to account for the following issues:

1. Unsplash API seems to throttle requests per second and so I created unsplashAPIcall.js to make the calls over a longer period of time. I then also had to alter MySQL and Riak filess to account for this.
2. Significant alterations made to MySQL files to add a bad, inefficient schema for comparison purposes to the original one I designed
3. For Riak, I had to lower the rate at which data was generated since Riak has a low limit for incoming querys

Hopefully everything is not all over the place as i suspect it is. Let me know if you need help deciphering what I did.

Thanks,
Nick

### CODE REVIEW

File: database-mongodb/MySQL/initiateSeeding.js [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 
 var connection = mysql.createConnection({
   host: mysqlHost,
   user: mysqlUser,
   password: mysqlPassword,
 });
 
-const table = 'CREATE TABLE IF NOT EXISTS SDC_Image_Service_MySQL.images (itemId VARCHAR(10) NOT NULL UNIQUE PRIMARY KEY, itemImages VARCHAR(90) NOT NULL)';
+const table = 'CREATE TABLE IF NOT EXISTS SDC_Image_Service_MySQL_10Million.images (itemId VARCHAR(10) NOT NULL UNIQUE PRIMARY KEY, itemImages VARCHAR(90) NOT NULL)';
+const badTable = 'CREATE TABLE IF NOT EXISTS SDC_Image_Service_MySQL_10Million.imagesBad (indexNumber INT NOT NULL UNIQUE PRIMARY KEY AUTO_INCREMENT, itemId VARCHAR(10) NOT NULL, itemImages VARCHAR(90) NOT NULL)';
```
Comment (cmonez) [inline database-mongodb/MySQL/initiateSeeding.js:12]: What about the badTable is bad necessarily? I see theres an indexNumber column in that table that doesn't exist in your original table. Is it just having the extra column?

File: database-mongodb/unsplashAPIcalls.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    .then(extractData)
+    .then(() => {
+      if (pageNumber === maxPagesToQuery && query === 'puppy') {
+        getUnsplashImages(1, urlsPerRequest, 'kitten');
+      } else if (pageNumber === maxPagesToQuery && query === 'kitten') {
+        const stringData = JSON.stringify(data);
+
+        const dataToWrite = `module.exports = JSON.parse('${stringData}')`;
+
+        fs.writeFileSync(`${__dirname}/unsplashData.js`, dataToWrite);
```
Comment (cmonez) [inline database-mongodb/unsplashAPIcalls.js:40]: Whats the point of blocking with writeFIleSync? Why not use the async version of this function? Everywhere i've read discourages using Sync functionality so i'm just curious about your choice here.

File: database-mongodb/unsplashAPIcalls.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        getUnsplashImages(pageNumber + 1, urlsPerRequest, query);
+      }
+    })
+    .catch(err => {
+      console.log(err);
+    });
+};
+
+const initiateUnsplashAPICall = function(pageNumber, urlsPerRequest, query) {
+  setTimeout(getUnsplashImages.bind(this, pageNumber, urlsPerRequest, query), 2000);
```
Comment (cmonez) [inline database-mongodb/unsplashAPIcalls.js]: Can you just explain to me a little bit what you're doing with the setTimeout call here.

File: README.md [hunk 4]
```diff
@@ -9,10 +9,10 @@
...
-4. You can now seed the database by running >npm run seed. If you also want to seed MySQL, run >npm run seedMySQL.
+3. In the app's root folder, there is a file titled "config.example.js". Make a duplicate of this file and rename it to "config.js". Then use the terminal to run >git status. You should not see the new config.js file listed. If you do, then the name of the file is wrong. Once the correct name of the file is confirmed, you can copy your Unsplash API key to the file. You can also update the other fields that require an update.
+4. Before seeding the database, you have to run >npm run preseed. This script will make 20 Unsplash API calls (which means you're allotment per hour will be charged 20) and will take some time to complete. Once that script finished running,  you can now seed the database by running >npm run seed. If you also want to seed MySQL, run >npm run seedMySQL. Same for Riak and >npm run Riak.
```
Comment (cmonez) [inline README.md]: Ahhh I think this answers the question I had below about using the sync writeFileSync?

**Review by cmonez — APPROVED** [review]
I liked reading the code for your unsplashAPIcalls file, especially how you use the extractData function in the promise chain. There were just a couple clarifying questions that I had.

File: README.md (hunk 4 above)
Comment (Nick-Zabalza) [reply README.md]: Yes, above is related to two of your previous questions.

**Pull request merged**